shopinvader_product_stock/models/product_product.py

```diff
@@ -9,13 +11,11 @@
 class ProductProduct(models.Model):
     _inherit = "product.product"
 
-    @api.multi
     @job(default_channel="root.search_engine.synchronize_stock")
     def _synchronize_all_binding_stock_level(self):
-        """
-        The goal of this function is to compute the new stock information
-        and update them in the data field. If data have change and the binding
-        is in done state we force it to 'to_update'.
+        """Compute new stock information and update index data.
+
+        If data changed and binding is in done state it forces it to 'to_update'.
         :return:
         """
         all_bindinds = self.mapped("shopinvader_bind_ids")
```

Comment:
I faced some issues when triggering an stock move confirmation from the sale portal, a sudo() should fix it...

Reply:
Indeed there's no issue w/ internal users (judging from ACL and RR and from our staging testing phase).
We can safely sudo here as the real operation is done w/ the backend user later.
Thanks for reporting. Fixed.